ui/components/MesheryMetrics/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import { styled } from "@mui/material/styles";
+import { PaperWithoutTitle } from "@/components/Paper";
+import { MetricsButton } from "@/components/Button"
+import AddIcon from '@mui/icons-material/Add';
+import {Typography} from "@mui/material";
+
+function MesheryMetrics(){
+
+    const CustomPaperWithoutTitle = styled(PaperWithoutTitle)(() => ({
```

Comment:
this `CustomPaperWithoutTitle` does not mean anything 😅 . For these kind of use cases, please use `sx` prop.

Reply:
Not that there's anything wrong with it, it's just that it makes it easy to understand.